setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# W391 blank line at end of file
+# E302 expected 2 blank lines, found 1
+ignore = W391,E302
```

Comment:
If a PR simply isn't merged until it passes an automated check it contributors will figure out the rest.